### PR TITLE
zh: add cdc document on eks/ecs

### PR DESCRIPTION
**Pull request by weekface**
### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: zh/deploy-on-alibaba-cloud.md [hunk 1]
```diff
@@ -189,10 +191,29 @@ category: how-to
...
 
+    如果要部署 TiCDC，可以在 db.yaml 中配置 `spec.ticdc`，例如：
+
+    ```yaml
+    spec
+      ...
+      ticdc:
+        baseImage: pingcap/ticdc
+        nodeSelector:
+          dedicated: TIDB_CLUSTER_NAME-ticdc
```
Comment (DanielZhangQD) [inline zh/deploy-on-alibaba-cloud.md:202]:
```suggestion
          dedicated: TIDB_CLUSTER_NAME-cdc
```

File: zh/deploy-on-alibaba-cloud.md [hunk 2]
```diff
@@ -189,10 +191,29 @@ category: how-to
...
+        replicas: 3
+        tolerations:
+        - effect: NoSchedule
+          key: dedicated
+          operator: Equal
+          value: TIDB_CLUSTER_NAME-ticdc
```
Comment (DanielZhangQD) [inline zh/deploy-on-alibaba-cloud.md:208]:
```suggestion
          value: TIDB_CLUSTER_NAME-cdc
```

File: zh/deploy-on-aws-eks.md [hunk 3]
```diff
@@ -186,10 +188,29 @@ region = us-west-21
...
 
+    如果要部署 TiCDC，可以在 db.yaml 中配置 `spec.ticdc`，例如：
+
+    ```yaml
+    spec:
+      ...
+      tiflash:
+        baseImage: pingcap/ticdc
+        nodeSelector:
+          dedicated: CLUSTER_NAME-ticdc
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:199]:
```suggestion
          dedicated: CLUSTER_NAME-cdc
```

File: zh/deploy-on-aws-eks.md [hunk 4]
```diff
@@ -186,10 +188,29 @@ region = us-west-21
...
+        replicas: 3
+        tolerations:
+        - effect: NoSchedule
+          key: dedicated
+          operator: Equal
+          value: CLUSTER_NAME-ticdc
```
Comment (DanielZhangQD) [inline zh/deploy-on-aws-eks.md:205]:
```suggestion
          value: CLUSTER_NAME-cdc
```

**Comment by ran-huang** [comment]
@DanielZhangQD PTAL.

**Pull request merged**